api/notd/manager.py

```diff
@@ -220,7 +221,8 @@ async def get_collection_daily_activities(self, address: str) -> List[Collection
                         minimumValue = min(minimumValue, collectionActivity.minimumValue) if minimumValue > 0 else collectionActivity.minimumValue
                         maximumValue = max(maximumValue, collectionActivity.maximumValue)
                     transferCount += collectionActivity.transferCount
-            collectionActivitiesPerDay.append(CollectionDailyActivity(date=currentDate,transferCount=transferCount,saleCount=saleCount,totalValue=totalValue,minimumValue=minimumValue,maximumValue=maximumValue,averageValue=0))
+                    averageValue += totalValue / transferCount
```

Comment:
its still wrong


look how it is in the collection_activity_processor please and just copy from there